lib/elixir/lib/calendar/date_range.ex

```diff
@@ -38,13 +46,30 @@ defmodule Date.Range do
       {:ok, false}
     end
 
-    def count(%Date.Range{first_in_iso_days: first_in_iso_days, last_in_iso_days: last_in_iso_days}) do
+    def count(%Date.Range{
+          first_in_iso_days: first_in_iso_days,
+          last_in_iso_days: last_in_iso_days
+        }) do
       {:ok, abs(first_in_iso_days - last_in_iso_days) + 1}
     end
 
-    def reduce(%Date.Range{first_in_iso_days: first_in_iso_days, last_in_iso_days: last_in_iso_days,
-                           first: %{calendar: calendar}}, acc, fun) do
-      reduce(first_in_iso_days, last_in_iso_days, acc, fun, calendar, first_in_iso_days <= last_in_iso_days)
+    def reduce(
+          %Date.Range{
+            first_in_iso_days: first_in_iso_days,
+            last_in_iso_days: last_in_iso_days,
+            first: %{calendar: calendar}
+          },
+          acc,
+          fun
+        ) do
+      reduce(
+        first_in_iso_days,
+        last_in_iso_days,
```

Comment:
I considered shortening the `*_in_iso_days` bound variable names in the `reduce/3` function to just `*_days`, which would cut down the `reduce/5` call to a one-liner but decided against it. The formatter keeps the code airy and light.